orbit/pkg/packaging/linux_shared.go

```diff
@@ -228,7 +228,7 @@ CPUQuota=20%
 [Install]
 WantedBy=multi-user.target
 `),
-		constant.DefaultFileMode,
+		constant.DefaultSystemdUnitMode,
```

Comment:
Nice one! I vaguely recall that this would result in a warning somewhere.

Reply:
Yes, it's actually kind of serious because on Debian systems it makes apt returns non-zero when upgrading fleet-osquery, which breaks config management tools like saltstack, etc.

The warning is:

` systemd[1]: Configuration file /lib/systemd/system/orbit.service is marked world-inaccessible. This has no effect as configuration data is accessible via APIs without restrictions. Proceeding anyway.`

Reply:
Thanks for fixing this @nunesvn. You rock!